tests/pylint/runpylint.py

```diff
@@ -1,52 +1,122 @@
 #!/usr/bin/python3
 
+import atexit
+import shutil
 import sys
+import tempfile
+import os
 
-from pocketlint import FalsePositive, PocketLintConfig, PocketLinter
+from os import path
 
+from censorship import CensorshipConfig, CensorshipLinter, FalsePositive
 
-class BlivetLintConfig(PocketLintConfig):
 
+class BlivetLintConfig(CensorshipConfig):
     def __init__(self):
-        PocketLintConfig.__init__(self)
-
-        self.falsePositives = [FalsePositive(r"Catching an exception which doesn't inherit from (BaseException|Exception): (BlockDev|DM|Crypto|Swap|LVM|Btrfs|MDRaid|Utils|G)Error$"),
-                               FalsePositive(r"Instance of 'int' has no .* member"),
-                               FalsePositive(r"Method 'do_task' is abstract in class 'Task' but is not overridden"),
-                               FalsePositive(r"Method 'do_task' is abstract in class 'UnimplementedTask' but is not overridden"),
-                               FalsePositive(r"No value for argument 'member_count' in unbound method call$"),
-                               FalsePositive(r"No value for argument 'smallest_member_size' in unbound method call$"),
-                               FalsePositive(r"Parameters differ from overridden 'do_task' method$"),
-                               FalsePositive(r"Bad option value '(subprocess-popen-preexec-fn|try-except-raise|environment-modify)'"),
-                               FalsePositive(r"Instance of '(Action.*Device|Action.*Format|Action.*Member|Device|DeviceAction|DeviceFormat|Event|ObjectID|PartitionDevice|StorageDevice|BTRFS.*Device|LoopDevice)' has no 'id' member$"),
-                               FalsePositive(r"Instance of 'GError' has no 'message' member")  # overriding currently broken local pylint disable
-                               ]
+        super().__init__()
```

Comment:
I think you could just omit `__init__` if all it does it call `super().__init()`.